frappe/commands/site.py

```diff
@@ -329,6 +329,29 @@ def set_admin_password(context, admin_password):
 		finally:
 			frappe.destroy()
 
+
+@click.command('set-limit')
+@click.option('--site', help='site name')
+@click.argument('limit', type=click.Choice(['email', 'space', 'user']))
+@click.argument('value')
+@pass_context
+def set_limit(context, site, limit, value):
+	"Set user / space / email limit for a site. Set as none for no limit."
+	from frappe.commands import get_site
+	from frappe.limits import set_limits, get_limits
+	if not site:
+		site = get_site(context)
+	with frappe.init_site(site):
+		limit = limit+'_limit'
+		if value.lower() == 'none':
```

Comment:
This seems an ugly way to do it 😒. Let's have another function `clear_limit` to clear a limit.